backend/pkg/api/dashboard.go

```diff
@@ -158,7 +158,7 @@ func NewDashboardResponse(response *http.Response) (*DashboardResponse, error) {
 }
 
 func NewDashboard(authConfig *auth.AuthConfig, uuid string, from string, to string, datasourceID int) (*Dashboard, error) {
-	url := "http://" + authConfig.AuthString() + "localhost:3000/api/dashboards/uid/" + uuid
+	url := "http://" + authConfig.AuthString() + authConfig.URL + "/api/dashboards/uid/" + uuid
```

Comment:
Would this be better done using an `Authorization` header? I think URL encoding auth strings has been deprecated in the HTTP standard (not 100% about this, but I recall reading it).

Reply:
It's probably the worst way to do it and any other way would be better, yup